core-aam/core-aam.html

```diff
@@ -3397,12 +3392,13 @@
 		<section>
 			<h2>Substantive changes since the <a href="http://www.w3.org/TR/2015/WD-core-aam-1.1-20151119/">last public working draft</a></h2>
 			<ul>
+				<li>14-Mar-2017: Undid the 1.1 addition of RELATION_POPUP_FOR to ATK/AT-SPI mappings of aria-haspopup. While the addition is correct in terms of platform expectations, the ARIA spec provides no means through which user agents could reliably implement this.</li>
 				<li>20-Jan-2017: Added AX API mappings for aria-colcount, aria-colspan, aria-colindex, aria-rowcount, aria-rowindex, and aria-rowspan.</li>
 				<li>07-Dec-2016: Added UIA mappings for roles feed, figure, and term, and for aria-colcount, aria-colspan, aria-colindex, aria-rowcount, aria-rowindex, aria-rowspan, and aria-details.  Added UIA Range Value Pattern for range related roles scrollbar, slider, and spinbutton.</li>
 				<li>24-Oct-2016: Changed name of IA2 object property name for aria-placeholder to "placeholder-text" (was "placeholder").</li>
 				<li>04-Oct-2016: Add reverse relationships for aria-details and aria-errormessage on platforms MSAA/IA2 and ATK/ATSPI</li>
 				<li>03-Oct-2016: Added aria-roledescription mappings for empty and whitespace strings as well as error handling text.</li>
-				<li>12-Sep-2016: Added RELATION_POPUP_FOR to ATK/AT-SPI mappings of aria-haspopup, for ARIA 1.1.</li>
```

Comment:
@Joanmarie  I think this removal is okay since the comment was never part of a published working draft.  The latest TR is:
https://www.w3.org/TR/core-aam-1.1/#changelog

But, just in case, @michael-n-cooper: is removal of a previously noted change allowed?